script/Ciel/Day 9/09_01_CIEL09_6.txt

```diff
@@ -130,8 +130,7 @@ If that's where that sentence was headed,
 [sha:764a383bcf5ecf7b3ceb3f2aedc3409f74574d8f]{
 -- Page 547, Offset 23542.
 -- 「わたしたち、いま絶交中なんです。けんかしてるんです。
-// 絶交 was 'never want to see you again' in 9_2B, but they do be seeing each other here,
-// so changed to not on speaking terms because she's not listening to him
+// Go look through at 絶交 usage
```

Comment:
I'm not a big fan of "broke up" since that so specifically means "end a romantic relationship" in english as far as I've encountered it and 絶交 is for _any_ relationship afaict. I might just condense this into:
> We're still in the middle of our falling-out, remember.